src/Tribe/Views/V2/View.php

```diff
@@ -587,24 +596,19 @@ public function setup_the_loop( array $args = [] ) {
 
 		$this->global_backup = [
 			'wp_query'  => $wp_query,
+			'$_SERVER' => isset($_SERVER) ? $_SERVER : [],
```

Comment:
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: There must be a single space before the last parenthesis

<sub>hash: d4c33f9261d1cd67e450f687ee5b92bf</sub>